vhdl-ext-nav.el

```diff
@@ -144,6 +144,7 @@ command.")
 Configuration should be done so that `vhdl-ext-navigation-ag-rg-hook' is run
 after the search has been done."
   (interactive)
+  (setq vhdl-ext-jump-to-parent-entity-starting-windows (length(window-list)))
```

Comment:
To avoid compiler warnings, the variable first needs to be declared with `defvar` before being assigned a value with `setq`, e.g:
```elisp
(defvar vhdl-ext-jump-to-parent-trigger nil
  "Variable to run the post ag/rg command hook.
Run only when the ag/rg search was triggered by `vhdl-ext-jump-to-parent-entity'
command.")
(defvar vhdl-ext-jump-to-parent-entity-starting-windows nil)
```

You can then assign it a value inside `vhdl-ext-jump-to-parent-entity`, where the rest of ag/rg variables are updated:
```elisp
    ;; Update variables used by the ag/rg search finished hooks
    (setq vhdl-ext-jump-to-parent-module-name entity-name)
    (setq vhdl-ext-jump-to-parent-module-dir proj-dir)
    (setq vhdl-ext-jump-to-parent-entity-starting-windows (length (window-list)))
```